### DISCUSSION TITLE
How do we use ADHOC Filter with Infinity?

### DISCUSSION THREAD

**Discussion by krgeron** [grafana/grafana-infinity-datasource#65] [Q&A]
I need to have these values reflected to my dashboard so I can dynamically filter the results.

![image](https://user-images.githubusercontent.com/29052475/116503768-72b14a80-a8e9-11eb-82f2-387bba701bcb.png)

**Comment by yesoreyeram** [comment]
You can refer the variables in the url or post body by $varname . Example http://example.com?assigned_to=$assignee

Have you tried? What error you are getting?
